code/procs/stat_logging.dm

```diff
@@ -163,6 +163,26 @@
 				if (M.master)
 					var/mob/mymaster = ckey_to_mob(M.master)
 					if (mymaster) special = mymaster.real_name
+			if (ROLE_FLOCKMIND)
+				var/relay_successful = FALSE
+				if (isflockmob(M.current))
+					if (!istype(M.current, /mob/living/critter/flock/drone))
+						var/mob/living/intangible/flock/flockmind/flockmind = M.current
+						relay_successful = flockmind.flock.relay_finished
+					else
+						var/mob/living/critter/flock/drone/flockdrone = M.current
+						relay_successful = flockdrone.flock.relay_finished
+				special = "Relay transmission [relay_successful ? "successful" : "unsuccessful"]"
+			if (ROLE_FLOCKTRACE)
+				if (isflockmob(M.current))
+					var/datum/flock/flock_joined = null
+					if (!istype(M.current, /mob/living/critter/flock/drone))
+						var/mob/living/intangible/flock/trace/flocktrace = M.current
+						flock_joined = flocktrace.flock
+					else
+						var/mob/living/critter/flock/drone/flockdrone = M.current
+						flock_joined = flockdrone.flock
+					special = "Part of Flock [flock_joined.name]"
```

Comment:
Flocktraces are already listed under their respective flockminds in the end chatlog

Reply:
This was saved to the "hublog" which I assumed was sent to some logging site?